### PR TITLE
WT-4968 remove all committed and connection level commit timestamps

### PR DESCRIPTION
**Pull request by boboeee**
Removed the all committed and connection level commit timestamps, as they are now deprecated because of durable history initiative. all_durable timestamp was created in https://github.com/wiredtiger/wiredtiger/pull/4737.

### CODE REVIEW

**Comment by wtbuild** [comment]
autobuild: Can one of the admins verify this patch?

**Comment by tetsuo-cpp** [comment]
autobuild: test this please

**Comment by tetsuo-cpp** [comment]
retest this please

**Comment by boboeee** [comment]
finish retest

File: test/suite/test_timestamp14.py [hunk 1]
```diff
@@ -269,26 +253,20 @@ def test_all_durable(self):
...
         cur1[1] = 1
         session1.commit_transaction('commit_timestamp=3')
-        self.assertTimestampsEqual(
-            self.conn.query_timestamp('get=all_committed'), '3')
         self.assertTimestampsEqual(
             self.conn.query_timestamp('get=all_durable'), '3')
 
         # We have a running transaction with a lower commit_timestamp than we've
-        # seen before. So all_durable (like all_committed) should return (lowest
+        # seen before. So all_durable (like all_durable) should return (lowest
```
Comment (tetsuo-cpp) [inline test/suite/test_timestamp14.py]: Nit: Can remove `(like all_durable)`.

File: test/suite/test_timestamp14.py [hunk 2]
```diff
@@ -43,35 +43,31 @@ class test_timestamp14(wttest.WiredTigerTestCase, suite_subprocess):
     uri = 'table:' + tablename
     session_config = 'isolation=snapshot'
 
-    def test_all_committed(self):
+    def test_all_durable_old(self):
```
Comment (tetsuo-cpp) [inline test/suite/test_timestamp14.py]: 😎

Comment (boboeee) [reply test/suite/test_timestamp14.py]: :)

File: test/suite/test_timestamp14.py (hunk 1 above)
Comment (boboeee) [reply test/suite/test_timestamp14.py]: Got It! Changed.

**Comment by sueloverso** [comment]
Was this supposed to be merged into the durable history branch instead of develop? Because this is now breaking all kinds of MongoDB tests. See BF-15701.

**Pull request merged**